filebeat/harvester/reader/timeout.go

```diff
@@ -64,6 +64,6 @@ func (p *Timeout) Next() (Message, error) {
 		}
 		return msg.line, msg.err
 	case <-time.After(p.timeout):
-		return Message{}, p.signal
+		return NewMessage(), p.signal
```

Comment:
ouch, here we allocate a throwaway message + common.MapStr just to ignore it. Zero value of message should suffice here.